include/cripts/Context.hpp

```diff
@@ -100,8 +100,8 @@ class Context
 } // namespace Cript
 
 // This may be weird, but oh well for now.
-#define get()               _get(context)
-#define set(_value)         _set(context, _value)
+#define Get()               _get(context)
```

Comment:
Can we do this?
```c
#define Response::get()    Response::_get(context)
```

Reply:
No, C++ doesn't allow this (I tried that long ago, it's a good idea, but no bueno).

```
/tmp/foo.cc:3:17: warning: ISO C99 requires whitespace after the macro name [-Wc99-extensions]
#define Response::get() Response::_get(context)
                ^
1 warning generated.
```